File: Models/Entities/Donghua.cs
namespace DonghuaFlix.Models.Entities
{
    public class Donghua
    {
        public int DonghuaId { get; set; }
        public string Title { get; set; }
        public string TitleOriginal { get; set; }
        public string Description { get; set; }
        public string studio { get; set; }
        public string coverImage { get; set; }
        public DateTime releaseDate { get; set; }
        public DonghuaStatus status { get; set; }
        public List<Genre> genres { get; set; }
        public List<Episode> episodes { get; set; }

    }
}

File: Frontend/donghuaflixfront/src/Domain/entities/donghua.ts
import { DonghuaStatus } from "../enum/donghuaStatus"
import { DonghuaType } from "../enum/donghuaType"
import { Genre } from "../enum/genre"

export interface Donghua {
    DonghuaId: string
    Title: string
    titleEnglish: string
    description: string
    banner: string
    rating: number
    Sinopse: string
    Studio: string
    ReleaseYear: Date
    Genre: string[]
    Type: DonghuaType
    Status: DonghuaStatus
    Image: string | null
  

  }